docker-compose.yml

```diff
@@ -14,6 +14,7 @@ web:
   environment:
     PORT: 8070
     DATABASE_URL: postgres://postgres@db:5432/postgres
+    DEBUG:
```

Comment:
Cool, I set it to True in docker-compose.yml and set it to default to false in the settings.py file.